Makefile

```diff
@@ -7,9 +7,8 @@ PHP=$(shell which php)
 JOBS=$(shell nproc)
 
 # PHP CS Fixer
-PHP_CS_FIXER=vendor/bin/php-cs-fixer
-PHP_CS_FIXER_ARGS=--diff --diff-format=udiff --verbose
-export PHP_CS_FIXER_IGNORE_ENV=1
+PHP_CS_FIXER=./.tools/php-cs-fixer
+PHP_CS_FIXER_URL="https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v3.2.1/php-cs-fixer.phar"
```

Comment:
php-cs-fixer is not compatible with symfony 6 yet.

waiting https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6095 to be merged and new release